static/js/containers/HomePage.js

```diff
@@ -22,52 +24,35 @@ class HomePage extends React.Component {
     frontpage: RestState<Array<string>>,
     posts: RestState<Map<string, Post>>,
     subscribedChannels: RestState<Array<string>>,
-    channels: RestState<Map<string, Channel>>
+    channels: RestState<Map<string, Channel>>,
+    showSidebar: boolean
   }
 
   componentWillMount() {
     this.fetchFrontpage()
   }
 
   fetchFrontpage = () => {
-    const { dispatch, subscribedChannels } = this.props
+    const { dispatch } = this.props
 
     dispatch(actions.frontpage.get()).then(posts => {
       dispatch(setPostData(posts))
     })
-
-    if (!subscribedChannels.loaded && !subscribedChannels.processing) {
-      dispatch(actions.subscribedChannels.get()).then(channels => {
-        dispatch(setChannelData(channels))
-      })
-    }
   }
 
   render() {
-    const { posts, frontpage, subscribedChannels, channels, dispatch } = this.props
+    const { posts, frontpage, dispatch } = this.props
     const dispatchableToggleUpvote = toggleUpvote(dispatch)
 
     return (
-      <div className="triple-column">
-        <div className="first-column">
-          <SubscriptionsSidebar
-            // $FlowFixMe: flow thinks these might be undefined
-            subscribedChannels={safeBulkGet(subscribedChannels.data, channels.data)}
-          />
-        </div>
-        <div className="second-column">
-          <Card title="Home Page">
-            <PostList
-              // $FlowFixMe: flow thinks these might be undefined
-              posts={safeBulkGet(frontpage.data, posts.data)}
-              toggleUpvote={dispatchableToggleUpvote}
-              showChannelLinks={true}
-            />
-          </Card>
-        </div>
-        <div className="third-column" />
-        <br className="clear" />
-      </div>
+      <Card title="Home Page">
+        <PostList
+          // $FlowFixMe: flow thinks these might be undefined
+          posts={safeBulkGet(frontpage.data, posts.data)}
+          toggleUpvote={dispatchableToggleUpvote}
```

Comment:
Nit: I don't see why we shouldn't just write this as `toggleUpvote={toggleUpvote(dispatch)}` and avoid the variable. We do it in [PostPage](https://github.com/mitodl/open-discussions/pull/118/files#diff-c3c3ec36add7a1078a2c2d54a3b777bdR93)

Reply:
I don't disagree, but I didn't change this in this PR, it's just showing up in the diff as a side-effect of indentation level changes in the JSX